manifests/params.pp

```diff
@@ -386,6 +386,8 @@
   $proxy_snmptrapperfile                    = '/tmp/zabbix_traps.tmp'
   $proxy_sourceip                           = undef
   $proxy_sshkeylocation                     = undef
+  $proxy_sslcertlocation                    = '/usr/lib/zabbix/ssl/certs'
+  $proxy_sslkeylocation                     = '/usr/lib/zabbix/ssl/keys'
```

Comment:
I think zabbix doesn't have default value. That should be undef in this case.

Reply:
you are correct, will update